client/my-sites/customer-home/main.jsx

```diff
@@ -509,6 +510,33 @@ class Home extends Component {
 						</Card>
 					) }
 					{ ! siteIsUnlaunched && <StatsCard /> }
+					<Card className="customer-home__free-photo-library-card">
+						{ ! isMobile() && (
+							<video
+								className="customer-home__free-photo-library-demonstration"
+								controls
+								muted
+								preload="none"
+								poster={ freePhotoLibraryThumbnail }
+							>
+								<source
+									src="https://wpcom.files.wordpress.com/2020/02/free-photo-library-demonstration.mp4"
+									type="video/mp4"
+								/>
+							</video>
+						) }
+						<CardHeading>{ translate( 'Over 40,000 Free Photos' ) }</CardHeading>
+						<p className="customer-home__card-subheader">
+							{ translate(
+								'The WordPress.com Free Photo Library integrates ' +
+									'your site with beautiful copyright-free photos to ' +
+									'create stunning designs.'
+							) }
+						</p>
+						<Button href={ localizeUrl( 'https://support.wordpress.com/free-photo-library/' ) }>
```

Comment:
Linking to the support docs makes sense as a first step. @dezzie what do you think of an external icon for the button or link, similar to our support card?

<img width="347" alt="Screen Shot 2020-02-19 at 1 56 09 PM" src="https://user-images.githubusercontent.com/1270189/74880300-c99fb500-531f-11ea-879a-9cbb21a720b1.png">

Reply:
It's the `no-noninteractive-element-interactions `and the `no-noninteractive-tabindex` ones. Is there a way to fix them or should I just disable those two rules specifically?